### PR TITLE
Get activate real_time_update objects

### PR DESCRIPTION
**Pull request by azime**
Select all real_time_update objects by contributor, start and end date.

### CODE REVIEW

File: kirin/core/model.py [hunk 1]
```diff
@@ -192,3 +190,16 @@ def __init__(self, raw_data, connector,
         self.status = status
         self.error = error
         self.received_at = received_at
+
+    def __repr__(self):
+        return '<RealTimeUpdate %r>' % self.id
+
+    @classmethod
+    def all(cls, contributor, start_date=datetime.datetime.now().date(), end_date=None):
```
Comment (TeXitoi) [inline kirin/core/model.py]: Using now as default seems strange. A coherent default, for me, would be -infinity

File: kirin/core/model.py [hunk 2]
```diff
@@ -192,3 +190,16 @@ def __init__(self, raw_data, connector,
...
+        query = cls.query.filter_by(contributor=contributor)
+        query = query.join(cls.trip_updates)
```
Comment (kinnou02) [inline kirin/core/model.py]: All the indexes for this request exist?

File: kirin/core/model.py (hunk 1 above)
Comment (kinnou02) [inline kirin/core/model.py]: We will want more than one contributor

**Comment by azime** [comment]
Index added

**Pull request merged**